GVFS/GVFS.Common/Database/IPlaceholderDatabase.cs

```diff
@@ -4,8 +4,7 @@ namespace GVFS.Common.Database
 {
     public interface IPlaceholderDatabase
     {
-        int Count { get; }
-
+        int Count();

```

Comment:
Minor: Why was this changed from a property to a method?

If it stays a method could you update the name to include a verb (e.g. `GetCount`, `RetrieveCount`)

Reply:
>Why was this changed from a property to a method?

Because it is now running a SQL query and not simply returning a stored value.  Making it a method makes it more obvious that it is running some code and could take longer that retrieving a value.  This is why `IEnumerable` has a `Count()` method and that is why I named it `Count` to follow that precedent but I can rename it.